plaso/engine/single_process.py

```diff
@@ -49,10 +50,16 @@ def _ProcessPathSpec(self, extraction_worker, parser_mediator, path_spec):
       excluded_find_specs = (
           self.collection_filters_helper.excluded_file_system_find_specs)
 
+    # pylint: disable=try-except-raise
     try:
       extraction_worker.ProcessPathSpec(
           parser_mediator, path_spec, excluded_find_specs=excluded_find_specs)
 
+    # Raise on coding errors.
+    except (AttributeError, ImportError, NameError, TypeError,
```

Comment:
Please factor this tuple out a constant somewhere (lib/errors? lib/defintions?)

Reply:
done